### ISSUE TITLE
Components need a parent

### ISSUE THREAD

**Issue by eduard-vm** [ag-grid/ag-grid-vue#12]
Hi.
https://github.com/ag-grid/ag-grid-vue/blob/master/dist/vueComponentFactory.js#L216
Why is this commented? This solves many problems. What's the matter?
Thanks in advance.

**Comment by makinggoodsoftware** [comment]
Hi, Can you elaborate? Thanks

**Comment by eduard-vm** [comment]
Hi. Yes, of course. For example I implement cellRenderFramework in which there is a button. Pressing it calls an external method(a method of the wrapper of the table). 
I have the following solutions:
To use:
this.params.context.externalMethod() (if I define this method in the context)
this.params.extenralMethod() (if I define this method in cellRendererParams)

But it could do without such a definition, if I had to refer to the parent: this.$parent.externalMethod()

Studying the code I came across the following:
https://github.com/ag-grid/ag-grid-vue/blob/master/dist/vueComponentFactory.js#L216
I was curious: what is the reason for this decision?
